mp/game/momentum/resource/ui/leaderboards/times.res

```diff
@@ -281,8 +281,8 @@
 		"RoundedCorners"		"15"
 		"pin_corner_to_sibling"		"0"
 		"pin_to_sibling_corner"		"0"
-		"visible"		"1"
-		"enabled"		"1"
+		"visible"		"0"
+		"enabled"		"0"
```

Comment:
This is outside the scope of this branch, but because this panel actually doesn't do anything and the related change is small, I'll let it slide.

Please don't do things like this in the future. Keep your branches scoped to the panel you're working on, touching other things when absolutely necessary. It's how bugs are introduced.

Reply:
Noted. I tried to make it obvious in the post that I slipped it in there, and figured it was valid because it was on the same UI, but I'll keep that in mind for next time.